contributors/devel/development.md

```diff
@@ -141,7 +141,8 @@ development environment, please [set one up](http://golang.org/doc/code.html).
 | 1.8            | 1.8.3       |
 | 1.9            | 1.9.1       |
 | 1.10           | 1.9.1       |
-| 1.11+          | 1.10.2      |
+| 1.11, 1.12     | 1.10.2      |
+| 1.13+          | 1.11.1      |
```

Comment:
@rf232 Thanks for updating this!

- 1.12 used 1.10.4: https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.12.md#external-dependencies
- 1.13 used 1.11.2: https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.12.md#external-dependencies
- 1.13+ uses 1.11.4: https://github.com/kubernetes/kubernetes/pull/72084

Reply:
I based this on the minimum_go_version as updated in kubernetes/kubernetes#69386 

(https://github.com/kubernetes/kubernetes/pull/69386/files#diff-dac3f0d616ae45f98d45d6420ed1be65L363 for the exact diff)

But if you think documenting the used go version is a better option, I can change it.

Reply:
let's go with recommended here as @nikhita suggests. Thanks for updating these docs -- it's appreciated! 🍰

Reply:
done